code/modules/paperwork/paper.dm

```diff
@@ -570,11 +586,48 @@ var/global/datum/topic_state/default/paper_state/paper_topic_state = new
 // Crumpled Paper
 ///////////////////////////////////////////////////
 /obj/item/paper/crumpled
-	name       = "paper scrap"
-	icon_state = "scrap"
+	name        = "paper scrap"
+	is_crumpled = TRUE
+
+// Stub type for moving teleportation scrolls into a modpack.
+/obj/item/paper/scroll
```

Comment:
can you turn these into paper planes? do those turn back into normal paper? i would hope not